pkg/proxy/ipvs/proxier.go

```diff
@@ -659,21 +542,21 @@ func (proxier *Proxier) isInitialized() bool {
 
 // OnServiceAdd is called whenever creation of new service object is observed.
 func (proxier *Proxier) OnServiceAdd(service *api.Service) {
-	if proxier.serviceChanges.Update(nil, service, newServiceInfo) && proxier.isInitialized() {
+	if proxier.serviceChanges.Update(nil, service) && proxier.isInitialized() {
```

Comment:
Can we invoke proxier.OnServiceUpdate(nil, service)?

Same comment for the OnServiceDelete function.

Reply:
Done.